ksso/main.py

```diff
@@ -51,6 +52,56 @@
 logging.getLogger("ksso").setLevel(logging.WARNING)
 
 
+def logout(client_id: str, aws_role_arn: str) -> None:
+    """Logout and remove cached credentials."""
+    if bool(client_id) != bool(aws_role_arn):
+        logger.error("--client-id and --aws-role-arn must be specified together")
+        exit(1)
+
+    removed_credentials = credstore.logout(client_id, aws_role_arn)
+    if removed_credentials > 0:
+        if client_id and aws_role_arn:
+            print(f"Removed credentials for {client_id} and role {aws_role_arn}")
+        else:
+            print(f"Removed {removed_credentials} credential(s)")
+    else:
+        print("No cached credentials found to remove")
+    return
+
+
+def login(config: str, client_id: str, aws_role_arn: str) -> AWSCredentials:
+    """
```

Comment:
Pls specify docs in the same format as above

Reply:
Done